csharp/FileManager/Triggers/CreateFileSetTrigger.cs

```diff
@@ -0,0 +1,36 @@
+﻿using FileManager;
+using Interfaces;
+using Storage.Local;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FileManager
+{
+    class CreateFileSetTrigger : ITrigger<Context>
+    {
+        public void Action(Context arguments)
+        {
+            List<IFile> files = new();
+            for(int i =2; i < arguments.Args.Length-1; i+=2)

```

Comment:
What is about auto-formatting for readability?

Reply:
Btw if length can be very big, you should put in variable, because it will give a little perfomance boost.
`for(int i = 2, n = arguments.Args.Length; i < n; i += 2)`